lib/when_committed.rb

```diff
@@ -1,30 +1,42 @@
 require 'when_committed/version'
+require 'active_record'
 
-module WhenCommitted
-  module ActiveRecord
-    def self.included(base)
-      base.after_commit :run_when_committed_callbacks
-      base.after_rollback :clear_when_committed_callbacks
+module WhenCommitted::ActiveRecord
```

Comment:
Careful with this.

```
module WhenCommitted
  module ActiveRecord
```

is *not* exactly equivalent to

```
module WhenCommitted::ActiveRecord
```

You can probably get away with it here, but there are implications. I wouldn't change it unless you have a good reason. See https://cirw.in/blog/constant-lookup and http://stackoverflow.com/a/29437662